addons/packages/packetfence.spec

```diff
@@ -649,6 +649,11 @@ if [ ! -f /usr/local/pf/conf/pf_omapi_key ]; then
     /usr/bin/openssl rand -base64 -out /usr/local/pf/conf/pf_omapi_key 32
 fi
 
+# Create server local RADIUS secret
+if [ ! -f /usr/local/pf/conf/local_secret ]; then
+    date +%s | sha256sum | base64 | head -c 32 > /usr/local/pf/conf/local_secret
```

Comment:
Mmm, fair point.
We might have to review the whole permissions scheme.
I'll let this one slide for now.
We'll revisit the issue.